### ISSUE TITLE
List of str is incorrectly prepped using json.dumps on newer Python version

### ISSUE THREAD

**Issue by melvinkcx** [wharton/drf-excel#84]
If you have a list of strings as value, the XLSXListField incorrectly preps it using `json.dumps()`.

I believe this is not an intended behaviour, judging from the code comments. 

This probably has to do with the fact that `str` is also an `Iterable` in newer versions of Python.

![Screenshot 2024-06-21 at 3 01 14 PM](https://github.com/wharton/drf-excel/assets/16914545/105cdcec-fdff-4379-9a06-ad00d2aefefb)

![Screenshot 2024-06-21 at 3 05 58 PM](https://github.com/wharton/drf-excel/assets/16914545/3acfe0f9-b156-4319-94ff-5f7a531dbd07)

One potential solution that I found is:

https://stackoverflow.com/a/44328500/10955067

---

## Workaround

Use `xlsx_custom_mappings`:

```python
xlsx_custom_mappings = {
        col: lambda value: ", ".join(map(str, value))
        for col in [
            "col_1", "col_2", "col_3
        ]
    }
```

**Comment by rptmat57** [comment]
good catch

**Comment by melvinkcx** [comment]
Thanks for the quick fix! Would it be possible to get it released soon?

**Comment by FlipperPA** [comment]
@melvinkcx @rptmat57 I can actually get this published today, since I'm at the Wagtail Space sprints.

**Comment by FlipperPA** [comment]
@rptmat57 @melvinkcx I've published as version `2.4.1` with a few bug fixes and... a long overdue upgrade to `pyproject.toml`!

**Comment by melvinkcx** [comment]
Thanks for the quick action, much appreciated!
